shardingsphere-proxy/shardingsphere-proxy-common/src/main/java/org/apache/shardingsphere/proxy/config/ShardingConfigurationLoader.java

```diff
@@ -54,8 +53,8 @@
      */
     public ShardingConfiguration load(final String path) throws IOException {
         Collection<String> schemaNames = new HashSet<>();
-        YamlProxyServerConfiguration serverConfig = loadServerConfiguration(getResourceFile(path + "/" + SERVER_CONFIG_FILE));
-        File configPath = getResourceFile(path);
```

Comment:
why remove this, the bug is caused by script, this is usable when run the proxy in idea and load the absolute path